### PR TITLE
Revised digital book chapters

### PR DESCRIPTION
**Pull request by FSchwendinger**
Performed a global check of digital book chapters 0 to 7 as discussed per e-mail.

Checklist before merging:

- [NA] Existing tests still work (check by running the test suite, e.g. from RStudio).
- [NA] Added tests (if you added functionality) or fixed existing test (if you fixed a bug).
- [NA] Clean code has been attempted, e.g. intuitive object names and no code redundancy.
- [X] Documentation updated:
  - [ ] Function documentation
  - [X] Chapter vignettes for GitHub IO
  - [ ] Vignettes for CRAN
- [NA] Corresponding issue tagged in PR message. If no issue exist, please create an issue and tag it.
- [ ] Updated release notes in `inst/NEWS.Rd` with a user-readable summary. Please, include references to relevant issues or PR discussions.
- [ ] Added your name to the contributors lists in the `DESCRIPTION` file, if you think you made a significant contribution.
- [NA] GGIR parameters were added/removed. If yes, please also complete checklist below.

### CODE REVIEW

File: vignettes/chapter0_GetStarted.Rmd [hunk 1]
```diff
@@ -23,13 +23,15 @@ knitr::opts_chunk$set(echo = TRUE)
 This section will guide you through the process of configuring your work
 environment to initiate the use of GGIR. For this purpose, you will need
 at least one accelerometer file to be processed by GGIR, please feel
-free to download [this .gt3x file](https://www.dropbox.com/s/covfyj31sarfaw0/GGIR01.gt3x?dl=0) or [this .cwa file](https://biobank.ndph.ox.ac.uk/ukb/refer.cgi?id=131620)
+free to download [this .gt3x
+file](https://www.dropbox.com/s/covfyj31sarfaw0/GGIR01.gt3x?dl=0) or
+[this .cwa file](https://biobank.ndph.ox.ac.uk/ukb/refer.cgi?id=131620)
```
Comment (vincentvanhees) [inline vignettes/chapter0_GetStarted.Rmd:28]: Why are lines in many places in this PR split across multiple lines? It is not practical to have sentences split by line breaks everywhere, because when we revise the text we need to also revise all the line breaks. so, I think it is best to just keep sentences on a single line, where rendering in html will take care of how it fills up the page.

File: vignettes/chapter0_Installation.Rmd [hunk 2]
```diff
@@ -30,7 +30,7 @@ knitr::opts_chunk$set(echo = TRUE)
 ## Install GGIR
 
 Install the latest released version of GGIR with its dependencies from
-CRAN. You can do this with one command from the console command line:
+CRAN. You can do this with one command in your RStudio console:
```
Comment (vincentvanhees) [inline vignettes/chapter0_Installation.Rmd:33]: Not everyone who uses GGIR uses RStudio, because on clusters we usually use R directly. Maybe R command line is the best way of phrasing it.

File: vignettes/chapter0_Support.Rmd [hunk 3]
```diff
@@ -42,58 +42,59 @@ library(GGIR)
 
 ## Question and problems
 
-For general questions and issues please report in the [GGIR google
-group](https://groups.google.com/forum/#!forum/RpackageGGIR)[.]{.underline}
+For general questions and issues please visit the [GGIR google
+group](https://groups.google.com/forum/#!forum/RpackageGGIR) and create
+a new conversation[.]{.underline}
```
Comment (vincentvanhees) [inline vignettes/chapter0_Support.Rmd:47]: thread instead of conversation

File: vignettes/chapter0_Support.Rmd [hunk 4]
```diff
@@ -42,58 +42,59 @@ library(GGIR)
...
 -   The [**Extra GGIR support
     session**](https://www.accelting.com/ggir-training/#:~:text=GGIR%20private%20training%22/%3E-,Extra%20GGIR%20support%20session,-Aim%3A%20Single%20session),
-    a single session aimed to answer specific questions or address the
-    challenges you may run into.
+    a single session aimed at answering specific questions or addressing
+    the challenges you may run into.
 
 ## Dedicated support
 
-If you need dedicated support with the use of GGIR, want GGIR to be modified or enhanced in line to your needs, then please contact [Vincent van Hees](https://www.accelting.com/).
```
Comment (vincentvanhees) [inline vignettes/chapter0_Support.Rmd:99]: please avoid splitting to multiple lines.

File: vignettes/chapter1_WhatIsGGIR.Rmd [hunk 5]
```diff
@@ -3,8 +3,11 @@ title: "1. What is GGIR"
 output: rmarkdown::html_vignette
 vignette: >
   %\VignetteIndexEntry{1. What is GGIR}
-  %\VignetteEngine{knitr::rmarkdown}
   %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+editor_options: 
+  markdown: 
+    wrap: sentence
```
Comment (vincentvanhees) [inline vignettes/chapter1_WhatIsGGIR.Rmd:10]: why is this?

File: vignettes/chapter2_Pipeline.Rmd [hunk 6]
```diff
@@ -30,14 +29,14 @@ GGIR, has a computational structure of six parts which are applied sequentially
...
+-   **Part 3:** Estimation of sustained inactivity bouts (rest periods) and sleep, needed as input for Part 4.
+-   **Part 4:** Labels the sustained inactive periods derived in Part 3 as sleep, or sustained inactivity during day time, per night and per file.
+-   **Part 5:** Compiles time series with classification of sleep and physical activity intensity categories by re-using information derived in part 2, 3 and 4. Next, Part 5 generates a descriptive summary metrics such as total time spent in intensity categories, the number of bouts, time spent in bouts, average acceleration per category, and fragmentation.
+-   **Part 6:** Facilitates analyses that rely on direct access to the full recording such as household co-analysis and circadian rhythm analysis. The reason why GGIR is split up in parts is that it avoids having to re-run all preceding parts if you only want to make a small change in the more downstream parts. The specific order and content of the parts have evolved over time, with Parts 1 and 2 developed between 2011 and 2013, Parts 3 and 4 between 2013 and 2015, Part 5 from 2017 to 2020, and Part 6 from 2023 to 2024.
```
Comment (vincentvanhees) [inline vignettes/chapter2_Pipeline.Rmd:35]: These revisions are not correct. I intentionally used the word create and not developed, because the development is still ongoing. I want to emphasize the initial creation phase. If that is confusing then maybe refer to these time periods as when the parts were "introduced"?

**Review by vincentvanhees — CHANGES_REQUESTED** [review]
Thanks @FSchwendinger I have had a quick look at the first half of the changed files and left some comments below. I will probably have more time to look at the other half on Friday or next week.

File: vignettes/chapter0_GetStarted.Rmd (hunk 1 above)
Comment (FSchwendinger) [reply vignettes/chapter0_GetStarted.Rmd:28]: This is unintentional and seems to be caused by using visual mode to revise the text. Will make sure to revise this.

File: vignettes/chapter0_Installation.Rmd (hunk 2 above)
Comment (FSchwendinger) [reply vignettes/chapter0_Installation.Rmd:33]: Agree, will revise this.

File: vignettes/chapter0_Support.Rmd (hunk 3 above)
Comment (FSchwendinger) [reply vignettes/chapter0_Support.Rmd:47]: will revise this.

File: vignettes/chapter1_WhatIsGGIR.Rmd (hunk 5 above)
Comment (FSchwendinger) [reply vignettes/chapter1_WhatIsGGIR.Rmd:10]: again, this is unintentional. Will revise this.

File: vignettes/chapter2_Pipeline.Rmd (hunk 6 above)
Comment (FSchwendinger) [reply vignettes/chapter2_Pipeline.Rmd:35]: Alright, will use "create" instead.

File: vignettes/chapter7_DescribingDataWithoutKnowingSleep.Rmd [hunk 7]
```diff
@@ -33,59 +33,59 @@ Average acceleration is known to be correlated with the activity-related energy
...
+In the specific approach, as proposed by [Rowlands et al.](https://pubmed.ncbi.nlm.nih.gov/31808014/), these quantiles are referred to as the **MX metrics**.
 The MX metrics should not be confused with the most active continuous X hours, e.g.
 M10, as used in circadian rhythm research that also can be derived with GGIR (see parameter `winhr`).
 
-To use the MX metrics as proposed by Rowlands et al., specify the durations of the 24h day that you wish to filter out the accelerations for.
-For example, to generate the minimum acceleration value for the most active 30 minutes, you can call `qlevels = (1410/1440)`, which will filter out the lowest 1410 minutes of the day.
-This can also be used as a nested term to generate multiple metrics, for example to call M60, M30 and M10, you can use the following parameter:
+To use the MX metrics as proposed by Rowlands et al., specify the durations of the 24h day during which you want to filter out the accelerations values.
+For example, to generate the minimum acceleration value for the most active accumulated 30 minutes, you can call `qlevels = (1410/1440)`, which will filter out the lowest 1410 minutes of the day.
```
Comment (vincentvanhees) [inline vignettes/chapter7_DescribingDataWithoutKnowingSleep.Rmd:60]: Please rephrase:
"For example, to generate the minimum acceleration value for the most active accumulated 30 minutes, you can call `qlevels = (1410/1440)`, which will filter out the lowest 1410 minutes of the day."

to

"For example, to generate the minimum acceleration value for the most active accumulated 30 minutes, you can call `qlevels = (1410/1440)`, which will identify the acceleration corresponding to the most active 30 minutes in a 1440 minute day."

File: vignettes/chapter7_DescribingDataWithoutKnowingSleep.Rmd [hunk 8]
```diff
@@ -33,59 +33,59 @@ Average acceleration is known to be correlated with the activity-related energy
...
+This parameter also accepts nested term to generate multiple MX metrics. For example, to call M60, M30, and M10, you can specify the following:
 
 `qlevels = c(c(1380/1440), c(1410/1440), c(1430/1440))`.
 
-Note: if parameter `qwindow` has values less than 24 h, e.g. the school day (e.g. [Fairclough et al 2020](https://pubmed.ncbi.nlm.nih.gov/31593604/)), the denominator should be changed from 1440 (24h) to the new `qwindow` window length, e.g. if an 8-hour window is used then the denominator would be 480 rather than 1440 and the numerator would be 480 minus for example 60, 30, 10 or 5.
+Note: If parameter `qwindow` has values below 24 h, e.g. the school day (e.g. [Fairclough et al 2020](https://pubmed.ncbi.nlm.nih.gov/31593604/)), the denominator should be changed from 1440 (24h) to the new `qwindow` window length. If for instance an 8-hour window is used, then the denominator would be 480 rather than 1440 and the numerator would be 480 minus for example 60, 30, 10, or 5.
```
Comment (vincentvanhees) [inline vignettes/chapter7_DescribingDataWithoutKnowingSleep.Rmd:65]: qwindow can be used to segment the day in many different segment lengths, by which there is no single but multiple window lengths. Discussing the new window length makes this unnecessarily confusing.

Can you rephrase such that it is clear that this only works for one window length and that GGIR currently does not facilitate using a standardised duration for multiple unequal day segments.

File: vignettes/chapter7_DescribingDataWithoutKnowingSleep.Rmd [hunk 9]
```diff
@@ -33,59 +33,59 @@ Average acceleration is known to be correlated with the activity-related energy
...
+The quantile,s as discussed above, can be used to describe the accelerations that participants exceed in their most active “X” accumulated minutes in a day.
+In the specific approach, as proposed by [Rowlands et al.](https://pubmed.ncbi.nlm.nih.gov/31808014/), these quantiles are referred to as the **MX metrics**.
 The MX metrics should not be confused with the most active continuous X hours, e.g.
 M10, as used in circadian rhythm research that also can be derived with GGIR (see parameter `winhr`).
 
-To use the MX metrics as proposed by Rowlands et al., specify the durations of the 24h day that you wish to filter out the accelerations for.
-For example, to generate the minimum acceleration value for the most active 30 minutes, you can call `qlevels = (1410/1440)`, which will filter out the lowest 1410 minutes of the day.
-This can also be used as a nested term to generate multiple metrics, for example to call M60, M30 and M10, you can use the following parameter:
+To use the MX metrics as proposed by Rowlands et al., specify the durations of the 24h day during which you want to filter out the accelerations values.
```
[the remark this answers is not in the document]
Comment (vincentvanhees) [reply vignettes/chapter7_DescribingDataWithoutKnowingSleep.Rmd:59]: sorry, correction: Instead of what I asked above, can we replace the word 'filter out' by 'identify'?
I think the verb 'filter' makes it unclear.

File: vignettes/chapter7_DescribingDataWithoutKnowingSleep.Rmd (hunk 7 above)
Comment (FSchwendinger) [reply vignettes/chapter7_DescribingDataWithoutKnowingSleep.Rmd]: Agree, revise this accordingly.

File: vignettes/chapter7_DescribingDataWithoutKnowingSleep.Rmd (hunk 9 above)
[the remark this answers is not in the document]
Comment (FSchwendinger) [reply vignettes/chapter7_DescribingDataWithoutKnowingSleep.Rmd]: Done

File: vignettes/chapter7_DescribingDataWithoutKnowingSleep.Rmd [hunk 10]
```diff
@@ -33,59 +33,59 @@ Average acceleration is known to be correlated with the activity-related energy
...
-In the physical activity literature the distribution of acceleration values is often referred to as intensity distribution.
+The distribution of acceleration values is often referred to as intensity distribution in the physical activity literature.
 
 ## Derived descriptives
 
 ### Sets of quantiles (MX metrics by [Rowlands et al.](https://pubmed.ncbi.nlm.nih.gov/29360664/))
 
-The quantiles as discussed above can be used to describe the accelerations that participants exceed in their most active “X” accumulated minutes in a day.
-In the specific approach as proposed by [Rowlands et al.](https://pubmed.ncbi.nlm.nih.gov/31808014/) these quantiles are referred to as the **MX metrics**.
+The quantile,s as discussed above, can be used to describe the accelerations that participants exceed in their most active “X” accumulated minutes in a day.
```
[the remark this answers is not in the document]
Comment (FSchwendinger) [reply vignettes/chapter7_DescribingDataWithoutKnowingSleep.Rmd]: I corrected this.

File: vignettes/chapter6_DataImputation.Rmd [hunk 11]
```diff
@@ -16,18 +16,18 @@ knitr::opts_chunk$set(
...
-For most of the subsequent analysis done by GGIR the imputed time series is used, while for a few analyses the time series without invalid segments are used:
+All time segments classified as non-wear or clipping (see Chapter 3) and those masked by the study protocol (see Chapter 5) are treated as invalid data.
+In GGIR part 2, the epoch level metric values are imputed, while a log is kept of which epochs were imputed.
+For most of the subsequent analysis done by GGIR, the imputed time series is used. The time series without invalid segments are used only for a few analyses:
 
 -   Weighted average of full recording
 
 -   Cosinor analysis (see Chapter 10)
 
 -   Optionally, sleep analysis (see Chapter 8)
```
[the remark this answers is not in the document]
Comment (FSchwendinger) [reply vignettes/chapter6_DataImputation.Rmd]: Yes, this makes it probably clearer. Revised this accordingly.

File: vignettes/chapter4_AccMetrics.Rmd [hunk 12]
```diff
@@ -108,7 +108,7 @@ GGIR(do.enmo = TRUE, do.mad = TRUE, do.bfen = TRUE, …)
 
 ### Approach to removing the gravitational signal component
 
-The table above indicates the approach used to separate the gravitation component.
+The table above indicates the approach used to separate the gravitation component from the actively produced acceleration.
```
[the remark this answers is not in the document]
Comment (FSchwendinger) [reply vignettes/chapter4_AccMetrics.Rmd]: Done.

File: vignettes/chapter3_QualityAssessment.Rmd [hunk 13]
```diff
@@ -14,55 +14,55 @@ knitr::opts_chunk$set(
 )
 ```
 
-Data quality assurance is at the core of GGIR. GGIR undertakes several key tasks aimed at evaluating, where possible enhancing the quality of the data, and documenting how data was enhanced. These tasks include time gap identification and imputation, automatic calibration  of the sensors, identification of periods when sensors were not worn , and scrutiny for segments of the signal exhibiting sustained, unusually high accelerations that are rarely encountered in real-world scenarios. 
+Data quality assurance is fundamental to GGIR's operations. GGIR undertakes several key tasks aimed at evaluating and where possible enhancing the quality of the data, as well as documenting how data was enhanced. These tasks include time gap identification and imputation, automatic calibration  of the sensors, identification of periods when sensors were not worn, and scrutiny for segments of the signal exhibiting sustained, unusually high accelerations that are rarely encountered in real-world scenarios.
```
[the remark this answers is not in the document]
Comment (FSchwendinger) [reply vignettes/chapter3_QualityAssessment.Rmd]: Done.

**Review by vincentvanhees — APPROVED** [review]
